Error during `make import`

I have the following error when I do `make import`:

```
File "/Users/aurelien/softwares/entitycore/app/cli/import_data.py", line 109, in get_or_create_annotation_body
    if annotation_body["@id"] == "nsg:Neuron":
       ~~~~~~~~~~~~~~~^^^^^^^
KeyError: '@id'
make: *** [import] Error 1
```

@eleftherioszisis  I think your PR from today added the line at which entitycore breaks

--- a/app/cli/curate.py
+++ b/app/cli/curate.py
@@ -1,5 +1,7 @@
 import datetime
 
+from app.logger import L
+
 BRAIN_REGION_REPLACEMENTS = {
     "Accessory supraoptic group: Other": "2218808594",
     "Agranular insular area, dorsal part, layer 2": "3250982806",
@@ -753,5 +755,9 @@ def curate_distribution(distribution, project_context):
     assert distribution["contentSize"]["value"] > 0
     assert distribution["digest"]["algorithm"] == "SHA-256"
     assert distribution["digest"]["value"] is not None
-    assert distribution["atLocation"]["@type"] == "Location"
+    if "atLocation" in distribution:
+        assert distribution["atLocation"]["@type"] == "Location"
+    else:
+        msg = f"Distribution had not atLocation: {distribution}"
+        L.warning(msg)
     return distribution

--- a/app/cli/import_data.py
+++ b/app/cli/import_data.py
@@ -106,7 +106,7 @@ def get_or_create_annotation_body(annotation_body, db):
 
     if annotation_type is MTypeClass:
         annotation_body = curate.curate_mtype(annotation_body)
-        if annotation_body["@id"] == "nsg:Neuron":
+        if "@id" in annotation_body and annotation_body["@id"] == "nsg:Neuron":
             # Too generic type, nothing to do
             return annotation_type, None
 

